src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py

```diff
@@ -435,11 +437,94 @@ def ensure_container_insights_for_monitoring(
                     },
                 }
             )
+
```

Comment:
@pfrcks, changes looks good to me. just want to sure functionality works as expected. Did you verify enabling monitoring addon with syslog on existing and also as part of the cluster creation?

Reply:
yes verified with both existing and new